### PR TITLE
added pretty parameter

### PR DESCRIPTION
**Pull request by bjdmeest**
pretty RML is more pretty than ugly RML

### CODE REVIEW

File: lib/tools.js [hunk 1]
```diff
@@ -11,10 +11,15 @@ const JsonLdSerializerExt = require('@rdfjs/serializer-jsonld-ext');
 rdfExt.parsers = rdfExt.parsers || new rdfExt.Parsers();
 rdfExt.serializers = rdfExt.serializers || new rdfExt.Serializers();
 rdfExt.parsers['text/turtle'] = N3Parser;
-rdfExt.serializers['application/json+ld'] = new JsonLdSerializerExt({outputFormat: 'string'});
-const {isomorphic} = require("rdf-isomorphic");
+rdfExt.serializers['application/json+ld'] = new JsonLdSerializerExt({ outputFormat: 'string' });
+const { isomorphic } = require("rdf-isomorphic");
 const Logger = require('./logger');
 
+const ttl_read = require('@graphy/content.ttl.read');
```
Comment (pheyvaer) [inline lib/tools.js:18]: For consistency I would write `ttlRead`.

File: lib/tools.js [hunk 2]
```diff
@@ -11,10 +11,15 @@ const JsonLdSerializerExt = require('@rdfjs/serializer-jsonld-ext');
...
+const ttl_write = require('@graphy/content.ttl.write');
+const dataset = require('@graphy/memory.dataset.fast');
+const { Readable } = require("stream")
```
Comment (pheyvaer) [inline lib/tools.js:21]: Nit: use single quotes like for the other requires.

File: lib/tools.js [hunk 3]
```diff
@@ -90,13 +95,38 @@ function logReadable(triples, base = null, cb = (err, result) => {
...
   }
-  let writer = new N3.Writer({prefixes});
+  let writer = new N3.Writer({ prefixes });
   makeReadable(triples, writer);
   compareY2RFiles
   writer.end(cb);
 }
 
+/**
+ * Create a pretty-printed tring string from an ugly turtle string
```
Comment (pheyvaer) [inline lib/tools.js:105]: What is a "tring"? 😝

**Review by pheyvaer — CHANGES_REQUESTED** [review]

**Comment by pheyvaer** [comment]
@bjdmeest Do we need a test for the pretty part?

**Comment by bjdmeest** [comment]
> 
> 
> @bjdmeest Do we need a test for the pretty part?

added!

**Pull request merged**